webhooks/pod_webhook.go

```diff
@@ -58,13 +58,26 @@ func (m *PodMutator) Handle(ctx context.Context, req admission.Request) admissio
 	}
 
 	// Check enablement
+	enabled := false
 	val, ok := pod.GetAnnotations()["openfeature.dev"]
 	if ok {
-		if val != "enabled" {
-			m.Log.V(2).Info("openfeature.dev Annotation is not enabled")
-			return admission.Allowed("openfeature is disabled")
+		m.Log.V(1).Info("DEPRECATED: The openfeature.dev annotation has been superseded by the openfeature.dev/enabled annotation. " +
+			"Docs: https://github.com/open-feature/open-feature-operator/docs/annotations")
```

Comment:
This isn't the right URL.

Reply:
updated to `https://github.com/open-feature/open-feature-operator/blob/main/docs/annotations`